tox.ini

```diff
@@ -28,4 +28,9 @@ basepython = python3.10
 deps =
     mypy==0.910
 
-commands = mypy --ignore-missing-imports --no-warn-no-return zyte_common_items tests
+commands = mypy \
+    --check-untyped-defs \
```

Comment:
`--check-untyped-defs` may cause a bit higher maintenance overhead, not sure. But let's try it; we can revert later if it's a burden.